includes/functions.php

```diff
@@ -2609,7 +2601,7 @@ function tsml_compare_imported_meeting($local_meeting, $import_meeting)
     $diff_fields = array();
     foreach($compare_fields as $field) {
         if ($normalized_meetings[0][$field] !== $normalized_meetings[1][$field]) {
-            $diff_fields[] = $field;
+            $diff_fields[] = $tsml_export_columns[$field];
```

Comment:
think we need an `array_key_exists` here in case the key isn't set for some reason

```
$diff_fields[] = array_key_exists($field, $tsml_export_columns) ? $tsml_export_columns[$field] : $field;
```

Reply:
All the keys and heading values are explicitly set in the Variables file. We remove some key/values when we create the $compare array to , but I don't see the $tsml_export_columns array getting touched again. So, by my reconning there's no way the key value could not be there. Are you sure about this or am I missing something?